suite-native/atoms/src/Sheet/BottomSheet.tsx

```diff
@@ -25,18 +18,26 @@ const sheetWrapperStyle = prepareNativeStyle<WrapperStyleProps>((utils, { insetB
     borderTopLeftRadius: utils.borders.radii.large,
     borderTopRightRadius: utils.borders.radii.large,
     paddingBottom: Math.max(insetBottom, utils.spacings.medium),
-    height: 600,
+    height: 'auto',
```

Comment:
This BottomSheet is used in more places than just TransactionDetail, this will probably break it in Settings > Localization. I suggest something like  `max-height: '70%'`.

Reply:
In Figma design, there is one bottom sheet height to fullscreen. https://www.figma.com/file/BlVnxNzUqrZkbhHgezLczh/%5BDesign-Handover%5D-MVP?node-id=9798%3A48281&t=BgjovEJwVvc7iwGu-4. `max-height: '70%'` would not allow this. Should I change this anyway?